bin/compile

```diff
@@ -276,8 +280,10 @@ fi
 mkdir -p "conf"
 cp "$basedir/util/autotune.php" "conf/autotune.php"
 cp "$basedir/../conf/nginx/base.conf.erb" "conf/nginx.conf.erb"
-# If PHP 7.3, custom config file to improve logging
-if [ "$(php_api_version)" = "20180731" ] ; then
+# Fix dirty logging on stdout/stderr (WARNING child with PID said on stderr). It
+# has been fixed in PHP 7.3 (https://github.com/php/php-src/pull/2458) with a
+# custom php-fpm parameter: `decorate_workers_output`
+if [[ "$(php_api_version)" > "${PHP_MODULE_API_VERSIONS["7.2"]}" ]] ; then
```

Comment:
Isn't it `-gt` ?

Reply:
`-gt` is for number

`>` works for string. it's a bash-ism

Reply:
But in shell, it's the same actually, [ "3" -gt "2" ] -> returns 0 :)

Reply:
Well both works then ^^ I'll switch to `-gt` as you seem more keen on this one